buildscript/deps.go

```diff
@@ -50,6 +55,44 @@ func requirePodman(ctx *task.Context) error {
 	return err
 }
 
+var podmanCurrentMachineRegexp = regexp.MustCompile(`currentmachine:\s*"?([^"\s]*)"?`)
+var podmanMachineStateRegexp = regexp.MustCompile(`machinestate:\s*"?([^"\s]*)"?`)
+
+func startPodmanMachine(ctx *task.Context) error {
+	out, err := sh.RunOutput(ctx, "podman", "machine", "info")
```

Comment:
I'm guessing this command takes a `--format json` argument which you could use instead of parsing the output with regexes.